server/bundler/css-hot-reload.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Module dependencies
+ */
+var os = require( 'os' ),
+	fs = require( 'fs' ),
+	path = require( 'path' ),
+	spawn = require( 'child_process' ).spawn,
+	debug = require( 'debug' )( 'css-hot-reload' ),
```

Comment:
same to here: https://github.com/Automattic/wp-calypso/pull/5510/files#r65801017